config/locales/devise.en.yml

```diff
@@ -9,10 +9,10 @@ en:
     failure:
       already_authenticated: "You are already signed in."
       inactive: "Your account is not activated yet."
-      invalid: "Invalid %{authentication_keys} or password."
+      invalid: " 电子邮箱／用户密码不正确."
```

Comment:
you don't want to modify `devise.en.yml`, instead should set Rails default locale to `zh-CN` and copy that file to `devise.zh-CN.yml` and make modifications there